dockerfile/rocm5.1.1-pytorch1.10.0.dockerfile

```diff
@@ -110,13 +110,14 @@ RUN cd /tmp && \
     cd /tmp && \
     rm -rf mlc
 
-# Install rccl with commitid 700b473
+# Install rccl with commitid 6707a27
+ARG NUM_MAKE_RCCL=2
 RUN cd /tmp && \
     git clone https://github.com/ROCmSoftwarePlatform/rccl.git && \
-    cd rccl && git checkout 700b473 && \
+    cd rccl && git checkout 6707a27 && \
     mkdir build && cd build && \
-    CXX=/opt/rocm/bin/hipcc cmake -DCMAKE_INSTALL_PREFIX=/opt/rocm/rccl .. && \
-    make -j ${NUM_MAKE_JOBS} && make install && \
+    CXX=/opt/rocm/bin/hipcc cmake -DCMAKE_INSTALL_PREFIX=/usr/local .. && \
+    make -j ${NUM_MAKE_RCCL} && make install && \
```

Comment:
I think without `-j` and `-j 2` do not have much difference here, you can remove it or hard code inline, the `NUM_MAKE_RCCL` is not adaptive

Reply:
ok, thanks